specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-05-15-preview/notebook.json

```diff
@@ -0,0 +1,461 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
+    "description": "Azure Cosmos DB Database Service Resource Provider REST API",
+    "version": "2022-05-15-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces": {
+      "get": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "description": "Gets the notebook workspace resources of an existing Cosmos DB account.",
+        "operationId": "NotebookWorkspaces_ListByDatabaseAccount",
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved notebook workspace resources.",
+            "schema": {
+              "$ref": "#/definitions/NotebookWorkspaceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-05-15-preview/cosmos-db.json#/definitions/ErrorResponse"
```

Comment:
>../../preview/2022-05-15-preview/cosmos-db.json [](http://example.com/codeflow?start=23&length=47)

Any reason you are using this path instead of ./cosmos-db.json?

Reply:
Seems like it was copied like that from the previous version (2022-02-15-preview). I have seen usage like that in various places, but none in the code that we are adding with the new version. Would you recommend we change them for this new version?